CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Added
   - Reimplement API to rely on SQLALchemy core functions.
   - Return error messages as 'error' property in result.
   - Add tearDown to the tests.
+  - support 'length=-1' to disable paging
```

Comment:
It's a keepachangelog format, so:
```
Unreleased
~~~~~~~~~
Added
~~~~~
  - Support `length=-1` to disable paging.
```

with the commit message always beginning with a capital letter, and an imperative verb and ending with a period, please.